bqplot/nbextension/Figure.js

```diff
@@ -22,7 +22,8 @@ define(["nbextensions/widgets/widgets/js/widget", "base/js/utils",
 
         initialize : function() {
             this.setElement(document.createElementNS(d3.ns.prefix.svg, "svg"));
-            this.el.classList.add("bqplot", "figure");
+            // Internet Explorer does not support classList for svg elements
+            this.el.setAttribute("class", "bqplot figure");
```

Comment:
Thanks. This does not seem to work on chrome, which complains that the `className` property only has a getter and no setter.